app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -2,6 +2,20 @@ module Spree
   module Admin
     class TaxjarSettingsController < Spree::Admin::BaseController
       def edit
+        @configuration = SuperGood::SolidusTaxjar.configuration
+      end
+
+      def update
+        configuration = SuperGood::SolidusTaxjar.configuration
+        configuration.update!(configuration_params)
+        flash[:success] = "TaxJar settings updated!"
```

Comment:
Should we use `update` here and test for the result before showing the flash message? If the update fails now it'll return a 500 error to the user.